packages/template-typescript-minimal/package.json

```diff
@@ -40,15 +43,230 @@
     "react-intl": "^5.25.1",
     "react-router-dom": "^5.3.4",
     "typescript": "4.9.5",
-    "zustand": "5.0.3"
+    "zustand": "5.0.3",
+
+    "@chakra-ui/skip-nav": "^2.0.15",
+    "@peculiar/webcrypto": "^1.4.2",
+    "@salesforce/commerce-sdk-react": "^3.2.0-extensibility-preview.0",
+    "@tanstack/react-query-devtools": "^4.29.1",
+    "card-validator": "^8.1.1",
+    "cross-fetch": "^3.1.4",
+    "express": "^4.19.2",
+    "focus-visible": "^5.2.0",
+    "helmet": "^4.6.0",
+    "lodash": "^4.17.21",
+    "nanoid": "^3.3.4",
+    "prop-types": "^15.8.1",
+    "query-string": "^7.1.3",
+    "raf": "^3.4.1"
   },
   "engines": {
     "node": "^16.11.0 || ^18.0.0 || ^20.0.0",
     "npm": "^8.0.0 || ^9.0.0 || ^10.0.0"
   },
   "mobify": {
     "app": {
-      "extensions": []
+      "extensions": [
+        [
+          "@salesforce/extension-url-mapping"
+        ],
```

Comment:
I forgot to mention this: I'm thinking that we can ask folks to install the url-mapping extension as the very 1st extension (and not as the last extension).

In my POC, the extension creates and saves the `Auth` object in `locals`, so that other extensions can reuse it if they need to call SCAPI. As a result, it makes more sense to have the extension as the very 1st one.